Terran-One/cw-simulate-ui
# WIP PR for Trace substep

## Issue link

[WL-XXX](https://trello.com/c/vlPLXss1/2-traverse-trace-for-the-ui)

## Description
Added the SubStepper component in order to render the trace.
I think it's getting rendered infinite times though, that needs to be fixed. (It's with dummy data as of now)
## Test steps

1. Go to simulation page
2. Do execute.
3. On stepper you can now see sub steps(through dummy data though)
